tools/feature_matrix.py

```diff
@@ -132,6 +141,12 @@ def enable_feature(feature, reason):
         setattr(settings, name, min_version)
 
 
+def disable_feature(feature):
+  """Allow the user to disable a feature that would otherwise be on by default.
+  """
+  disable_override_features.add(feature)
```

Comment:
Can we have the same default override behavior in enable and disable, so they are parallel?

Reply:
The behavior is asymmetric in that there is no implicit disabling, only implicit enabling. So disable_feature can only be an explicit disable and will always override. But enable_feature can be explicit for a feature requested by a user, or implicit for a feature enabled as a result of another feature bumping the min browser versions.